app/views/report/_export_widgets.html.haml

```diff
@@ -21,7 +21,7 @@
             %td.treegrid-node
               = item[:name]
             %td
-              %img{:src => status_icon_image_path(item[:exists])}
+              %i{:class => item[:exists] ? 'pficon pficon-blueprint' : 'pficon pficon-ok'}
```

Comment:
@skateman I think "pficon-warning-triangle-o" might get the point across better. What say you?

Reply:
:+1: for the clone